homeassistant/components/devolo_home_network/entity.py

```diff
@@ -53,6 +53,7 @@ def __init__(
             manufacturer="devolo",
             model=device.product,
             name=entry.title,
```

Comment:
btw, you can remove this line as devices without name default to the entry title